types/ember-data/test/adapter.ts

```diff
@@ -29,8 +29,14 @@ const AuthTokenHeader = DS.JSONAPIAdapter.extend({
     })
 });
 
-// Ensure that we are allowed to overwrite headers with a getter
+// Ensure that we are allowed to overwrite properties with a getter
 class GetterHeader extends DS.JSONAPIAdapter {
```

Comment:
Can you rename the class here since it's testing more than `headers` now? Maybe `GetterTest`?

Reply:
Thanks for the suggestion, done